packages/manager/apps/telecom/src/app/telecom/telephony/alias/configuration/ovhPabx/tts/tts.html

```diff
@@ -247,6 +249,7 @@ <h1 data-translate="telephony_alias_ovh_pabx_tts_title"></h1>
             <div
                 class="mt-2"
                 data-ng-if="$ctrl.tts.raw.length"
+                data-ng-show="$ctrl.tts.raw.length > itemsPerPage"
```

Comment:
Can't the two be grouped ?

Reply:
in this code, the pagination is generating the items (slicing the array) so adding the condition to the 'ng-if' will no work (the pagination won't be rendered and no items will be displayed)